libraries/standard/mqtt/src/iot_mqtt_validate.c

```diff
@@ -137,6 +172,8 @@ bool _IotMqtt_ValidateConnect( const IotMqttConnectInfo_t * pConnectInfo )
                          "the maximum supported length of %hu.",
```

Comment:
Nit: While you're touching this file, you might as well fix the "the" twice in a row on 171-172

Reply:
Fixed